src/vtk/ToolWidgetUtils/utils.ts

```diff
@@ -36,3 +36,19 @@ export function watchState(
     originalDelete();
   };
 }
+
+export const computeWorldCoords = (model: any) => (event: any) => {
+  const manipulator =
+    model.activeState?.getManipulator?.() ?? model.manipulator;
+  if (!manipulator) {
+    console.error('No manipulator');
+    return undefined;
+  }
+  const { worldCoords } = manipulator.handleEvent(
+    event,
+    model._apiSpecificRenderWindow
+  );
+  if (!worldCoords)
+    console.warn('Event cannot be converted to world coordinates');
```

Comment:
I think our current manipulators shouldn't trigger this unless there's an error case, so this is a good add. If we start getting them legitimately, then we can remove it.